src/app/user-profile/user-profile.component.ts

```diff
@@ -41,4 +41,10 @@ export class UserProfileComponent implements OnInit {
     } 
   }
 
+  // profileCreationLogin() {
```

Comment:
do we need this?

Reply:
being saved at this time for maybe potential future use, Ill add a comment